integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/DistanceSearchSearchableSortableIT.java

```diff
@@ -98,6 +98,31 @@ public void searchableNotSortable() {
 		assertThat( query ).hasDocRefHitsAnyOrder( INDEX_NAME, CHEZ_MARGOTTE_ID, IMOUTO_ID );
 	}
 
+	@Test
+	public void searchableDefSortable() {
```

Comment:
"def"? Does it mean "default" or "definition"? Maybe avoid that kind of shortcut?